worker-core/src/main/java/com/hpe/caf/worker/core/WorkerTaskImpl.java

```diff
@@ -109,7 +113,7 @@ public void setResponse(final WorkerResponse response) {
             response.getTaskStatus(), responseContext,
             response.getQueueReference(), taskMessage.getTracking(),
             new TaskSourceInfo(getWorkerName(responseMessageType), getWorkerVersion()));
-
+        responseMessage.setPriority(priorityManager.getResponsePriority(taskMessage));
```

Comment:
All of the other properties of the responseMessage were set in the constructor.  Surely this should also be another constructor parameter rather than a separate call afterward?  (Although I still don't understand why the priority needs to be in the task message at all)